winterfell/services/claimService.js

```diff
@@ -21,20 +21,16 @@ module.exports = ClaimService;
  * which requires evaluating the angular hideExpression attribute for
  * each field against the current responses.
  *
- * @param formId
+ * @param formName
  * @param data
  * @returns {{answerable: number, answered: number}}
  */
-module.exports.calculateProgress = function calculateProgress(formId, data) {
+function calculateProgress(formName, data) {
   var evaluate, i;
-  if (!(formId in formlyFields)) {
-    throw new Error("Unknown formId: " + formId);
-  }
-  var template = formlyFields[formId];
+  var template = formlyFields[formName];
```

Comment:
hmm I think we said:
```
formId: “VBA-21-0966-ARE”,
formTitle: "INTENT TO FILE A CLAIM FOR COMPENSATION AND/OR PENSION, OR SURVIVORS PENSION AND/OR DIC",
vfi: {
   title: "Intent to File"
   description: "..."
}
```

but then it's possible a form that is saved in db has an _id (row id). We could change `formId` to `formName`, but we'll need to differentiate that from `formTitle`